app/graphql/types/record_type.rb

```diff
@@ -146,48 +146,55 @@ def matched_phrases
 
     class RecordType < Types::BaseObject
       field :identifier, String, null: false, deprecation_reason: 'Use `timdex_record_id`'
-      field :timdex_record_id, ID, null: false
-      field :source, String, null: false
-      field :source_link, String, null: false
-      field :title, String, null: false
-      field :alternate_titles, [Types::AlternateTitleType], null: true
-      field :contributors, [Types::ContributorType], null: true
-      field :subjects, [Types::SubjectType], null: true
-      field :identifiers, [Types::IdentiferType], null: true
+      field :timdex_record_id, ID, null: false, description: 'TIMDEX unique identifier for the item'
+      field :source, String, null: false,
```

Comment:
This formatting doesn't come across as intended in our generated Reference Docs. I think a comma separated list without the line breaks might works best here.